publications-2022.bib

```diff
@@ -1,5 +1,89 @@
 % Encoding: US-ASCII
 
+@misc{africa2022lifex,
+    doi = {10.48550/ARXIV.2207.14668},
+    author = {Africa, Pasquale Claudio},
+    title = {lifex: a flexible, high performance library for the numerical solution of complex finite element problems},
+    publisher = {arXiv},
+    year = {2022}
+}
+
+@article{piersanti2022bivEM,
+    title={3D--0D closed-loop model for the simulation of cardiac biventricular electromechanics},
+    author={Piersanti, Roberto and Regazzoni, Francesco and Salvador, Matteo and Corno, Antonio F and Vergara, Christian and Quarteroni, Alfio and others},
+    journal={Computer Methods in Applied Mechanics and Engineering},
+    volume={391},
+    pages={114607},
+    year={2022},
+    publisher={Elsevier}
+}
+
+@article{vergara2022,
+    author = {Vergara, Christian and Stella, Simone and Maines, Massimiliano and Africa, Pasquale Claudio and Catanzariti, Domenico and Demattè, Cristina and Centonze, Maurizio and Nobile, Fabio and Quarteroni, Alfio and Del Greco, Maurizio},
+    title = {Computational electrophysiology of the coronary sinus branches based on electro-anatomical mapping for the prediction of the latest activated region},
+    journal = {Medical \& Biological Engineering \& Computing},
+    year = {2022},
+    doi = {10.1007/s11517-022-02610-3}
+}
+
+@article{zingaro2022,
+    author = {Zingaro, Alberto and Fumagalli, Ivan and Fedele, Marco and Africa, Pasquale Claudio and Dede', Luca and Quarteroni, Alfio and Corno, Antonio Francesco},
+    title = {A geometric multiscale model for the numerical simulation of blood flow in the human left heart},
+    journal = {Discrete and Continuous Dynamical Systems - S},
+    volume = {15},
+    number = {8},
+    pages = {2391-2427},
+    year = {2022},
+    doi = {10.3934/dcdss.2022052}
+}
+
+@article{regazzoni2022emcirculation,
+    author = {F. Regazzoni and M. Salvador and P.C. Africa and M. Fedele and L. Dedè and A. Quarteroni},
```

Comment:
The system complains about the non-ASCII character in Dede's name. Can you fix this?

Reply:
Done! Thanks.